source/_docs/metrics.md

```diff
@@ -8,7 +8,6 @@ tags: [performance, logs]
 2. Toggle available date ranges by clicking **Day**, **Week**, or **Month**:
   ![Charts for pages served and visits within the Metrics tool of the Site Dashboard](/source/docs/assets/images/dashboard/metrics-graphs.png)
 
```

Comment:
Preserving @alexfornuto's original comment in #3448: 
> Consider expanding this screenshot to show the nav bar on the left with **Metrics** selected. This will make the screenshot useful for both steps.